Project.toml

```diff
@@ -1,17 +1,20 @@
+authors = ["fpacaud <francoispacaud8@gmail.com>"]
 name = "CUSOLVERRF"
 uuid = "a8cc9031-bad2-4722-94f5-40deabb4245c"
-authors = ["fpacaud <francoispacaud8@gmail.com>"]
 version = "0.2.1"
 
+[compat]
+CEnum = "0.4"
```

Comment:
Why do we need CEnum?

Reply:
I have no idea how that sneaked in. I blame Copilot 🙂. Fixed.